src/ui/public/doc_table/doc_table.html

```diff
@@ -104,6 +104,11 @@
   <kbn-infinite-scroll ng-if="infiniteScroll" more="addRows"></kbn-infinite-scroll>
 </div>
 
+<div ng-if="!columns.length" class="table-vis-error">
+  <h2><i class="fa fa-meh-o"></i></h2>
```

Comment:
How about also changing that message while you're at it? But maybe double-check with @cjcenizal just in case I misunderstood something.